src/gsi/control_vectors.f90

```diff
@@ -113,7 +113,7 @@ module control_vectors
 public dot_product  
 public prt_control_norms, axpy, random_cv, setup_control_vectors, &
      write_cv, read_cv, inquire_cv, maxval, qdot_prod_sub, init_anacv, &
-     final_anacv,c2sset_flg
+     final_anacv,c2sset_flg,e2sset_flg
```

Comment:
@jderber-NOAA  ,Shall we add a comment for this new parameter?

Reply:
Not really a new parameter.  Initially set to true and then false after first call to ensctl2state_set so that ensctl2state_set is only called once.  Not specified externally.

Reply:
Yes. I see how it is used.  Maybe, a comment like " internal controlling parameter" would help?  but, it is fine for the current status.